internal/receiver/discoveryreceiver/config_test.go

```diff
@@ -40,7 +40,7 @@ func TestValidConfig(t *testing.T) {
 
 	assert.Equal(t, 1, len(configs.ToStringMap()))
 
-	cm, err := configs.Sub("discovery/discovery-name")
+	cm, err := configs.Sub("discovery")
```

Comment:
much cleaner.. always wondered why we named it like that on the config that was overwritten!